### DISCUSSION TITLE
How to get an outstanding issue that has a fix merged asap?

### DISCUSSION THREAD

**Discussion by dbcall2** [aws/aws-sdk-js#4492] [General]
Hi All,

This merge request: https://github.com/aws/aws-sdk-js/pull/4310 has been outstanding for ages now and resolves a port mapping property validation error in the ecs client. Currently ServiceConnect is completely unusable using the ecs task registration client as you cannot set the appProtocol field. 

Who can escalate the approval of this merge request so this issue can finally be resolved? 

Blessings,
David

**Comment by ajredniwja** [comment]
Hi there, the files changed in https://github.com/aws/aws-sdk-js/pull/4310 are generated differently by different service teams hence the changes cant be made on the SDK side. Looking at ECS clients [registerTaskDefinition](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/ECS.html#registerTaskDefinition-property), it has a `portMappings` field which accepts the `appProtocol`. If thats not what you are looking for can you open a feature request with all the information about where the new field is expected and I can take it to the service team if they can implement that. 

Thanks

**Reply by dbcall2** [reply]
Upgrading to the latest boto3 client seems to have fixed it. Previously I was receiving the exact same validation errors as in issue 4310. Thanks again for your reply!
